lib/Service/ModelStore.php

```diff
@@ -148,7 +148,7 @@ public function load(int $id): Estimator {
 	}
 
 	/**
-	 * @param Estimator|Persistable $estimator
+	 * @param Estimator $estimator (Must implement Persistable)
```

Comment:
what about Estimator&Persistable? https://psalm.dev/docs/annotating_code/type_syntax/intersection_types/

Reply:
Can be done in a follow up, but this causes many issues as all functions also need to return the correct type.